dataplane/dplanerc/interface.go

```diff
@@ -531,18 +539,34 @@ func (ni *Reconciler) reconcile(ctx context.Context, config *oc.Interface) {
 
 	intf := ocInterface{name: config.GetName(), subintf: 0}
 	state := ni.getOrCreateInterface(config.GetName())
-	data, ok := ni.ocInterfaceData[intf]
+	data := ni.ocInterfaceData[intf]
```

Comment:
I think we still need to check whether data is nil before passing it to reconcileEthernet?